src/Pulp/Args/Types.purs

```diff
@@ -90,10 +90,12 @@ directories = {
   name: Just "<dir:dir:...>",
   parser: \arg -> do
     paths <- token <|> argErr arg "Needs a directory argument."
-    let paths' = split (Pattern Path.delimiter) paths
+    let paths' = ignoreNull $ split (Pattern Path.delimiter) paths
     for_ paths' requireDirectory
     pure $ Just (toForeign paths')
-  }
+  } where
+    ignoreNull [""] = []
```

Comment:
Ok, I don't think we need to worry about that here ;) I think if we're going to allow `--include ''` we should probably also allow `--include foo,,bar` and treat it the same as `--include foo,bar`, so that's why I suggested it.